features/claims/advocate/scheme_sixteen/advocate_fixed_fee_claim_submit.feature

```diff
@@ -0,0 +1,54 @@
+@javascript
+Feature: Advocate tries to submit a fee scheme 16 claim for a Fixed fee (Appeal against conviction)
+
+  @fee_calc_vcr
+  Scenario: I create an Appeal against conviction claim, and use the back button to invalidate it
+
+    Given the current date is '2023-11-16'
+    And I am a signed in advocate
+    And I am on the 'Your claims' page
+
+    And I click 'Start a claim'
+    And I select the fee scheme 'Advocate final fee'
+    Then I should be on the new claim page
+
+    And I select the court 'Caernarfon'
+    And I select a case type of 'Appeal against conviction'
+    And I enter a case number of 'A20181234'
+    And I enter scheme 16 main hearing date
+
+    Then I click "Continue" in the claim form and move to the 'Defendant details' form page
+
+    And I enter defendant, scheme 16 representation order and MAAT reference
+    And I add another defendant, scheme 16 representation order and MAAT reference
+
+    Then I click "Continue" in the claim form
+    Then I click the link 'Back'
+    And I should be in the 'Defendant details' form page
+
+    Then I should see 'Defendant 1'
+    And I should see 'Defendant 2'
+    And I should see 2 representation orders
+
+    Then I click "Continue" in the claim form
+
+    Given I insert the VCR cassette 'features/claims/advocate/scheme_sixteen/fixed_fee_calculations'
```

Comment:
Curious, does this exist? I can't find it locally

Reply:
Good catch! I forgot to add the VCR files to my git (they were fresh new ones). I'll do that and get them pushed up